components/controls/__tests__/RadioToggle-test.tsx

```diff
@@ -29,10 +29,25 @@ it('renders', () => {
 it('calls onCheck', () => {
   const onCheck = jest.fn();
   const wrapper = shallow(getSample({ onCheck }));
-  change(wrapper.find('input[value="two"]'), 'two');
+  change(wrapper.find('input[value="second"]'), 'second');
   expect(onCheck).toBeCalledWith('two');
 });
 
+it('calls with number type', () => {
+  const onCheck = jest.fn();
+  const wrapper = shallow(
+    getSample({
```

Comment:
Since you are working on this, could you rename this to `shallowRender` and type correctly the props ? :smile: